specification/search/Azure.Search/models-service.tsp

```diff
@@ -0,0 +1,9176 @@
+     */
+    Brands: "brands",
+
+    /**
+     * Categories.
+     */
+    Categories: "categories",
+
+    /**
+     * Description.
+     */
+    Description: "description",
+
+    /**
+     * Visual features recognized as people faces.
+     */
+    Faces: "faces",
+
+    /**
+     * Visual features recognized as objects.
+     */
+    Objects: "objects",
+
+    /**
+     * Tags.
+     */
+    Tags: "tags",
+}
+
+/**
+ * A string indicating which domain-specific details to return.
+ */
+union ImageDetail {
+    string,
+
+    /**
+     * Details recognized as celebrities.
+     */
+    Celebrities: "celebrities",
+
+    /**
+     * Details recognized as landmarks.
+     */
+    Landmarks: "landmarks",
+}
+
+/**
+ * A string indicating what entity categories to return.
+ */
+union EntityCategory {
+    string,
+
+    /**
+     * Entities describing a physical location.
+     */
+    Location: "location",
+
+    /**
+     * Entities describing an organization.
+     */
+    Organization: "organization",
+
+    /**
+     * Entities describing a person.
+     */
+    Person: "person",
+
+    /**
+     * Entities describing a quantity.
+     */
+    Quantity: "quantity",
+
+    /**
+     * Entities describing a date and time.
+     */
+    Datetime: "datetime",
+
+    /**
+     * Entities describing a URL.
+     */
+    Url: "url",
+
+    /**
+     * Entities describing an email address.
+     */
+    Email: "email",
+}
+
+/**
+ * Deprecated. The language codes supported for input text by
+ * EntityRecognitionSkill.
+ */
+union EntityRecognitionSkillLanguage {
+    string,
+
+    /**
+     * Arabic
+     */
+    ar: "ar",
+
+    /**
+     * Czech
+     */
+    cs: "cs",
+
+    /**
+     * Chinese-Simplified
+     */
+    `zh-Hans`: "zh-Hans",
+
+    /**
+     * Chinese-Traditional
+     */
+    `zh-Hant`: "zh-Hant",
+
+    /**
+     * Danish
+     */
+    da: "da",
+
+    /**
+     * Dutch
+     */
+    nl: "nl",
+
+    /**
+     * English
+     */
+    en: "en",
+
+    /**
+     * Finnish
+     */
+    fi: "fi",
+
+    /**
+     * French
+     */
+    fr: "fr",
+
+    /**
+     * German
+     */
+    de: "de",
+
+    /**
+     * Greek
+     */
+    el: "el",
+
+    /**
+     * Hungarian
+     */
+    hu: "hu",
+
+    /**
+     * Italian
+     */
+    it: "it",
+
+    /**
+     * Japanese
+     */
+    ja: "ja",
+
+    /**
+     * Korean
+     */
+    ko: "ko",
+
+    /**
+     * Norwegian (Bokmaal)
+     */
+    no: "no",
+
+    /**
+     * Polish
+     */
+    pl: "pl",
+
+    /**
+     * Portuguese (Portugal)
+     */
+    `pt-PT`: "pt-PT",
+
+    /**
+     * Portuguese (Brazil)
+     */
+    `pt-BR`: "pt-BR",
+
+    /**
+     * Russian
+     */
+    ru: "ru",
+
+    /**
+     * Spanish
+     */
+    es: "es",
+
+    /**
+     * Swedish
+     */
+    sv: "sv",
+
+    /**
+     * Turkish
+     */
+    tr: "tr",
+}
+
+/**
+ * Deprecated. The language codes supported for input text by SentimentSkill.
+ */
+union SentimentSkillLanguage {
```

Comment:
If this is truly deprecated you could try using the `#deprecated` directive over this. https://typespec.io/docs/standard-library/built-in-decorators/#@deprecated

Reply:
I will let the service team to make the call.

Reply:
`SentimentSkillLanguage` is only used by `#Microsoft.Skills.Text.SentimentSkill` which has been deprecated. https://learn.microsoft.com/azure/search/cognitive-search-skill-sentiment

So, deprecating this seems to be the correct choice, but I too would leave this to the service team to decide.